Files/Views/MainPage.xaml.cs

```diff
@@ -106,6 +107,60 @@ protected override async void OnNavigatedTo(NavigationEventArgs eventArgs)
                         await AddNewTab(typeof(ModernShellPage), ResourceController.GetTranslation("NewTab"));
                     }
                 }
+                else if (navArgs == "tryRecoverTabs")
+                {
+                    if ((bool)ApplicationData.Current.LocalSettings.Values["tabsSaved"])
+                    {
+                        var tabFile = await ApplicationData.Current.TemporaryFolder.GetFileAsync("savedTabs");
+                        try
+                        {
+                            var tabsToRestore = await FileIO.ReadLinesAsync(tabFile);
+                            foreach (string path in tabsToRestore)
+                            {
+                                await AddNewTab(typeof(ModernShellPage), path);
+                            }
+
+                            var toastContent = new ToastContent()
+                            {
+                                Visual = new ToastVisual()
+                                {
+                                    BindingGeneric = new ToastBindingGeneric()
+                                    {
+                                        Children =
+                                        {
+                                            new AdaptiveText()
+                                            {
+                                                Text = "We've restored your tabs"
```

Comment:
Add translation.

Reply:
This notification has been removed since tab's are no longer restored.

Reply:
It looks like the code is still in here, can you remove the code or add the translations?